docs/architecture.md

```diff
@@ -21,52 +21,52 @@ This leads to constant development and DevOps/MLOps work.
 
-MLRun comprises of the following layers:
+MLRun comprises the following layers:
 * **Feature & Artifact Store** - Handle the ingestion, processing, metadata and storage of data and features across multiple repositories and technologies
 * **Elastic Serverless Runtimes** - Convert simple code to scalable and managed micro-services with 
-  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  .
+  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  
 * **ML Pipeline Automation** - Automated data preparation, model training & testing, 
-  deployment of production (real-time) pipelines, and end to end monitoring.
+  deployment of production (real-time) pipelines, and end-to-end monitoring
 * **Central Management** - Unified portal, UI, CLI, and SDK to manage the entire MLOps workflow which is 
-  accessible from everywhere. 
+  accessible everywhere
 
 <a id="basic-components"></a>
 ## Basic Components
 
 MLRun has the following main components that are used throughout the system:
 
-- <a id="def-project"></a>**Project** &mdash; a container for organizing all of your work on a particular activity.
-    Projects consist of metadata, source code, workflows, data and artifacts, models, triggers, and member management for user collaboration.
+- <a id="def-project"></a>**Project** &mdash; a container for organizing all of your work on a particular activity;
+    Projects consist of metadata, source code, workflows, data and artifacts, models, triggers, and member management for user collaboration
 
-- <a id="def-function"></a>**Function** &mdash; a software package with one or more methods and runtime-specific attributes (such as image, command, arguments, and environment).
+- <a id="def-function"></a>**Function** &mdash; a software package with one or more methods and runtime-specific attributes (such as image, command, arguments, and environment)
 
-- <a id="def-run"></a>**Run** &mdash; an object that contains information about an executed function.
-    The run object is created as a result of running a function, and contains the function attributes (such as arguments, inputs, and outputs), as well the execution status and results (including links to output artifacts).
+- <a id="def-run"></a>**Run** &mdash; an object that contains information about an executed function;
+    The run object is created as a result of running a function, and contains the function attributes (such as arguments, inputs, and outputs), as well the execution status and results (including links to output artifacts)
 
-- <a id="def-artifact"></a>**Artifact** &mdash; versioned data artifacts (such as data sets, files and models) that are produced or consumed by functions, runs, and workflows.
+- <a id="def-artifact"></a>**Artifact** &mdash; versioned data artifacts (such as data sets, files and models) that are produced or consumed by functions, runs, and workflows
```

Comment:
REJECT: Please revert the changes (see my review comment for the first components bullet).